guides/catalog/02 Creating and Managing Products.ipynb

```diff
@@ -34,120 +36,93 @@
+   "id": "afa69bb7-7442-4d4d-9466-a49d130e5bfd",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "product = Product()\n",
-    "product.id = pid\n",
-    "product.name = name\n",
-    "product.tags = [\"examples\"]\n",
-    "product.save()\n",
-    "product.id"
+    "org = dl.auth.Auth().payload[\"org\"]\n",
+    "org"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "9c2ac3cf-6f01-4b18-a290-ebcdb5acd028",
+   "id": "a5155b5d-14a8-4071-bc3a-7f00a514157e",
    "metadata": {},
    "source": [
-    "Every object has a read-only `created` attribute with the timestamp from when it was first saved:"
+    "Since you may have other users within your `Organization` who have run this same example, let's take it one step further and append our default `Namespace` to our `Product ID`. Each `Descartes Labs` account has a unique default `Namespace` ID:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "f3922796-cb1e-4f40-98d5-226b786d7022",
+   "id": "6a28f311-aed9-4f87-905f-b954b353423b",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "product.created"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "baf11668-eb3f-4684-b0f3-8b8006eae0a2",
-   "metadata": {},
-   "source": [
-    "`save()` saves the product to the catalog in the cloud. Note that you get to choose an id for your product but it must be unique within your organization (you get an exception if it’s not). This code example is assuming the user is in the “descarteslabs” organization. The id is prefixed with the organization id on save to enforce global uniqueness and uniqueness within an organization. If you are not part of an organization the prefix will be your unique user id. You can find this unique user id on your IAM page if you click on your name in the upper right."
+    "user_namespace = dl.auth.Auth().namespace\n",
+    "user_namespace"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "4fda3ccf-0b1d-48ca-94e4-a84cb48e3b77",
+   "id": "b631be2e-3f1e-45eb-9956-02471f0f0c2d",
    "metadata": {},
    "source": [
-    "There are a few more attributes that you can set (see the [`Product`](https://docs.descarteslabs.com/descarteslabs/catalog/docs/product.html#descarteslabs.catalog.Product) API reference). You can update the product to define the timespan that it covers. This is as simple as assigning attributes and then saving again:"
+    "_Note: You do not always need to pass your personal `Namespace` in practice_"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "44800dd8-68c7-47f0-b4ae-9ed178a9ce7c",
+   "id": "1829e262-a26e-494f-a9e2-65dc03ef7c26",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "product.start_datetime = \"2012-01-01\"\n",
-    "product.end_datetime = \"2015-01-01\"\n",
-    "product.save()\n",
-    "product.start_datetime"
+    "pid = f\"{org}:{user_namespace}:guide-example-product\"\n",
```

Comment:
I don't like this but am not sure if there's a better way to do it.

Reply:
definitely agree it's not ideal, more than open to alternative approach